crypto-lion/chains.json

```diff
@@ -106,7 +133,7 @@
       "address": "panaceavaloper1yr03eaz9ejndc4rt9244m6kcuva3ahfdkca377",
       "restake": {
         "address": "panacea102snugmpm8f9csr7y8me4tzj0cqxuu2muttgl0",
-        "run_time": "every 4 hour",
+        "run_time": "every week",
```

Comment:
Unfortunately restake is only setup for minimum once per day runs, this will break things. 

Suggest you either remove from this network entirely, or run once a day

Reply:
Ohh I see, thank you, did not know that.

Let me try making it once a day, please.